### PR TITLE
TODOs update: Notebook - tensors-variables.ipynb

### PR DESCRIPTION
**Pull request by swetasingh-tudip**
- Added the TODOs in notebook “tensors-variables.ipynb”.

### CODE REVIEW

**Comment by swetasingh-tudip** [comment]
@gwendolynstripling I have updated the sequencing of TODOs in the notebook. Please look into it and let me know if any other updates are required.
If everything is good to go, feel free to merge the PR.
- Thanks!

File: courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/tensors-variables.ipynb [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "\n",
```
Comment (gstripling) [inline courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/tensors-variables.ipynb:19]: <p>Change to TODO 3.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (swetasingh-tudip) [reply courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/tensors-variables.ipynb:19]: Hi @gwendolynstripling I have updated the sequencing of TODOs, feel free to merge the PR.
Thanks!

Comment (gstripling) [inline courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/tensors-variables.ipynb:19]: <p>Please add two spaces before the "Introduction" header?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/tensors-variables.ipynb:19]: <p>Nice!  Thanks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/tensors-variables.ipynb:19]: <p>Can you try moving the #Your code here after the dytype = ?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/tensors-variables.ipynb:19]: <p>The images are there and were rendering before the lab was moved from my branch to master.  The path to the images need to be changed.   Thanks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/tensors-variables.ipynb:19]: <p>Broken image link.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/tensors-variables.ipynb:19]: <p>Nice addition.  Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (swetasingh-tudip) [reply courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/tensors-variables.ipynb:21]: Updated the same in the notebook.

Comment (swetasingh-tudip) [reply courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/tensors-variables.ipynb:21]: Updated the same in the notebook.

Comment (swetasingh-tudip) [reply courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/tensors-variables.ipynb:21]: Updated the path to the images, all images are now rendering properly in the notebook.

Comment (swetasingh-tudip) [reply courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/tensors-variables.ipynb:21]: Updated the image path.

**Comment by swetasingh-tudip** [comment]
@gwendolynstripling implemented the above suggestions  in the notebook. 
Thank you!

**Comment by swetasingh-tudip** [comment]
@gwendolynstripling please review and merge the PR if you get a chance.

**Pull request merged**